### ISSUE TITLE
Generic Implementation

### ISSUE THREAD

**Issue by SrijaniBagchi** [xzenglab/KGNN#1]
Can I run this code for any other version of Drugbank datasets? It's not clear how the inputs are incorporated into the code.

**Comment by xiaomingaaa** [comment]
Of course, you can use any version of DrugBank, as long as it is processed into KG like this repository. In addition, you also need to map the drugs contained in the DDI dataset to the corresponding entity in the knowledge graph.
